package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@trex-arms/sqs",
-  "version": "1.3.1-0",
+  "version": "2.0.0",
   "description": "",
   "main": "index.mjs",
```

https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html#including-declarations-in-your-npm-package
```suggestion
  "main": "index.mjs",
  "types": "index.d.ts",
```